site/content/blog/opentitan-quickstart-guide-video.md

```diff
@@ -0,0 +1,13 @@
++++
+Description = ""
+date = "2019-11-26T9:00:00+00:00"
+title = "OpenTitan Quickstart Guide video"
```

Comment:
nit: consistent capitalization (and possibly a more exciting/actionable title, but I can't think of one either right now)